### PR TITLE
Drone Pickup Delay

### PR DESCRIPTION
**Pull request by ghost**
Delays drone pickup by humanoids by two seconds.
For issue #280.

### CODE REVIEW

**Comment by ghost** [comment]
Sorry about those merges. I'm illiterate with GitHub.

File: code/modules/mob/living/simple_animal/friendly/drone/interaction.dm [hunk 1]
```diff
@@ -75,6 +75,10 @@
 		if(buckled)
 			user << "<span class='warning'>[src] is buckled to the [buckled.name] and cannot be picked up!</span>"
 			return
+		if(picked_up == 1)
+			return
+		picked_up = 1 //prevents duplication of drone
+		sleep(20)
```
Comment (oisin100) [inline code/modules/mob/living/simple_animal/friendly/drone/interaction.dm]: Can we not use
`do_after(user, 50)`
here to make it show the timing animation?

Comment (X-TheDark) [inline code/modules/mob/living/simple_animal/friendly/drone/interaction.dm]: Yes, I believe this whole part can be replaced with

`user << "You start picking up the [src]."`
`if(!do_after(user, 20, target = user))`
-- `user << "You failed picking up the [src]."`

**Comment by DavidEdwards** [comment]
Firstly, you need to set your PR base branch to "dev" not "master". Also, please create a .yml file for a changelog.

**Comment by ShadowDeath6** [comment]
Still not merged, where's the drone love?

**Pull request merged**